string id_pareto_front
---
string type